src/DefaultBuilder/src/WebHost.cs

```diff
@@ -223,14 +223,14 @@ internal static void ConfigureWebDefaults(IWebHostBuilder builder)
             }
         });
 
-        UseKestrel(builder);
+        ConfigureWebDefaultsCore(builder);
```

Comment:
This looks like a real problem. Doesn't his mean that `WebHost.CreateDefaultBuilder()` and `Host.CreateDefaultBuilder().ConfigureWebHostDefaults(webHostBuilder => { })` now no longer support regex constraints? AddRouting has been added by default to these [since 3.0](https://github.com/dotnet/aspnetcore/pull/4358).

I think we should be calling AddRouting() here to add back the `RegexInlineRouteConstraint` to all the "default" builders. We should probably add something to WebHostTests.cs to verify regex still works without an explicit call to AddRouting().

Reply:
I've configuration for routing options in ```ConfigureWebDefaults(...)```. I don't think that the slim builder calls this so this should cover scenarios where people call ```CreateDefaultBuilder(...)``` on ```WebHost```. Thoughts?